blueman/bluez/Base.py

```diff
@@ -11,30 +11,24 @@
 
 class BaseMeta(GObjectMeta):
     def __call__(cls, *args: str, **kwargs: str) -> "Base":
```

Comment:
``__call__`` does not allow pep3102 notation so we indeed have to remove ``*args``. But then we get an ugly error, see below. Which looks very confusing to me.

```python
d1 = Device('/org/bluez/hci0/dev_3C_F7_2A_07_BF_E3')
Traceback (most recent call last):
  File "<pyshell#1>", line 1, in <module>
    d1 = Device('/org/bluez/hci0/dev_3C_F7_2A_07_BF_E3')
TypeError: __call__() takes 1 positional argument but 2 were given
```

If I leave ``*args`` in ``__call__`` and add the pep3102 notation to ``__init__`` we die horrible death like below.

```python
d1 = Device('/org/bluez/hci0/dev_3C_F7_2A_07_BF_E3')
Traceback (most recent call last):
  File "<pyshell#4>", line 1, in <module>
    d1 = Device('/org/bluez/hci0/dev_3C_F7_2A_07_BF_E3')
  File "/home/sander/repos/blueman/blueman/bluez/Base.py", line 24, in __call__
    path = cls._obj_path
AttributeError: type object 'Device' has no attribute '_obj_path'
```

I like raising a RuntimeError much better than this...

Reply:
`__call__() takes 1 positional argument but 2 were given` makes sense to me. If you don't like it, I'm fine with the explicit `*args` check, but in any case we should make clear what calls are expected by changing all `__init__(self, obj_path: str)` to `__init__(self, *, obj_path: str)`. (If we would not check `*args` before, this would give `__init__() takes 1 positional argument but 2 were given` on `super().__call__(*args, **kwargs)`.)

Side note: PEP 3102 works fine for `__call__` but is not suitable in our case as we do not have any named keyword arguments.

Reply:
OK, I'll drop it in ``__call__``.